src/applications/mhv-landing-page/tests/e2e/medical-records/temporary-landing-page.cypress.spec.js

```diff
@@ -4,6 +4,8 @@ import { defaultUser as user } from '../../../mocks/api/user';
 import ApiInitializer from '../utilities/ApiInitializer';
 
 describe(`${appName} -- transitional Medical Records page enabled`, () => {
+  const pageHeading = 'Medical records';
```

Comment:
Note: I cannot import the constant from the container as the import breaks Cypress.